### PR TITLE
feat(pwd-reset): configurable labels

### PR DESCRIPTION
**Pull request by bodnara**
* added password labels to configuration
* changed password help to map
* allowed service access for pwd reset

### CODE REVIEW

**Review by mattjoke — COMMENTED** [review]
There are some misspellings, probably from earlier version, otherwise I think it looks good!

**Comment by bodnara** [comment]
Translations were fixed. Also query parameters are now merged correctly after login.

**Review by mattjoke — APPROVED** [review]

**Comment by bodnara** [comment]
Marked this  PR as breaking change since it changed configuration.

File: apps/password-reset/src/app/pages/password-reset-page/password-reset-page.component.ts [hunk 1]
```diff
@@ -1,38 +1,48 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
 import { StoreService } from '@perun-web-apps/perun/services';
+import { PasswordAction, PasswordLabels } from '@perun-web-apps/perun/models';
```
Comment (HejdaJakub) [inline apps/password-reset/src/app/pages/password-reset-page/password-reset-page.component.ts:4]: 'PasswordLabels' is defined but never used
```suggestion
import { PasswordAction } from '@perun-web-apps/perun/models';
```

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
- There is one use case when we want to correctly handle also errors in response from the backend - not only check when I am typing but also catch errors after sending a request for reset/activation - there is an infinitive loader. NOT only for `PasswordStrengthException`, but also for `LoginNotExistsException`.

image
image

- I realized one more thing. There isn't a really good solution for how we parse mode from the URL - we parse any occurrence of a string in the whole URL - `this.mode = queryParams.includes('activation') ? 'activation' : 'reset';` - I know that the namespace like this probably doesn't exist (I hope), but we should parse it in some better way, something like this: `this.mode = parseQueryParams('mode', queryParams) === 'activation' ? 'activation' : 'reset';`
image

- I know that this is not a part of your task, but if I log in by `service-access` with queryParams, everything looks good. But if I refresh the page, the queryParams are lost and I am not even able to type them back manually.

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
- It looks like there is still a problem with query params when I am logged in with service-access.
- I am not sure if we have some specific error on the backend for the use case when the account is already activated (IMHO we have), but I get this message also if the backend error is `PasswordStrengthFailedException` so in this case we probably do NOT inform the user correctly. I am not sure if the solution is just to modify the instance config to present a universal message for errors during the account activation or if we want to somehow deal with these errors separately. 
image

**Review by HejdaJakub — COMMENTED** [review]
If we decide to fix `/service-access` and the `queryParams` in another PR, I will have just one last suggestoin / question (in the comment below).

**Pull request merged**